### ISSUE TITLE
Consolidate spacing and layout tokens?

### ISSUE THREAD

**Issue by michaelmccombie** [mozilla/protocol-tokens#36]
Our spacing and layout tokens have redundant values. We can reduce these redundancies by removing duplicate values or consolidating our spacing and layout tokens in to a single set. This would require a bunch of refactoring.

**Proposal 1**
```
  spacing-xs: '4px'
  spacing-sm: '8px'
  spacing-md: '16px'
  spacing-lg: '24px'
  spacing-xl: '32px'

  layout-xs: '48px'
  layout-sm: '64px'
  layout-md: '96px'
  layout-lg: '160px'

```

**Proposal 2**
```
  spacing-xs: '4px'
  spacing-sm: '8px'
  spacing-md: '16px'
  spacing-lg: '24px'
  spacing-xl: '32px'
  spacing-2xl: '48px'
  spacing-3xl: '64px'
  spacing-4xl: '96px'
  spacing-5xl: '160px'

```

**Comment by craigcook** [comment]
The original idea for the different units was inspired by (stolen from) Carbon. They also have some repetition but [do a decent job of explaining why they have two scales](https://www.carbondesignsystem.com/guidelines/spacing) and when/where/why to use them. We can think on it some more and decide if we want to go further along the same two scale path that Carbon has, but with the way we've used these units in Protocol so far I think consolidating to a single set of common `$spacing-` units (Proposal 2) might make more sense.

**Comment by stephaniehobson** [comment]
Fixed in #35
